packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -105,6 +105,7 @@ class Dropdown extends React.PureComponent {
     }
 
     componentDidMount() {
+        this.updateSelected(this.props.value);
```

Comment:
@negarn this only set the value once component is being mounted, if you use `componentdidupdate` , the select item can change after components render too.

Reply:
@Mahboobeh-binary no, that doesn't work, because this component has its own function to handle on change of item, I don't necessarily want to override that, let it handle what it's doing and just update the selected index after it decides.

Reply:
In that case if it’s just gonna be set once as a default value , we usually set it in component constructor. But still if someone bind an observable outside to this state and change it the state won’t change.